libcxx/test/libcxx/numerics/numarray/class.gslice.array/assert.get.pass.cpp

```diff
@@ -0,0 +1,49 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// REQUIRES: has-unix-headers
+// UNSUPPORTED: c++03
+// UNSUPPORTED: libcpp-hardening-mode=none
+// XFAIL: availability-verbose_abort-missing
```

Comment:
@var-const after rebasing this test started to unexpectedly pass today on [Apple back-deployment with hardening enabled](https://buildkite.com/llvm-project/libcxx-ci/builds/33077#_) is that an issue in this test or an issue in main?

Reply:
@mordante This should now be 
```cpp
// XFAIL: libcpp-hardening-mode=debug && availability-verbose_abort-missing
```
We no longer check the error message in hardening modes below debug (debug still calls verbose abort but production modes just trap), so the availability of verbose abort is no longer important for these tests other than in the debug mode.

Reply:
(I plan to soon add a lit feature that would combine all the requirements related to `check_assertion.h` together which should hopefully make this a bit nicer)